tsconfig.json

```diff
@@ -5,7 +5,7 @@
     "experimentalDecorators": true,
     "sourceMap": true
   },
-  "exclude": [
-    "node_modules"
+  "files": [
+  	"lib/index.ts"
```

Comment:
Well, if we use `exclude`, it parses all `.ts` files regardless if they're `import`ed or not.
This way it's following the `import` path from that entry file.

The dist works:
<img width="418" alt="screen shot 2015-11-30 at 12 32 08" src="https://cloud.githubusercontent.com/assets/480469/11469346/66dfc2b0-975e-11e5-920f-6f173d0d9fdb.png">